cumulusci/cli/ui.py

```diff
@@ -2,137 +2,78 @@
-
-        if wrap_cols:
-            self._table_wrapper(self._table, wrap_cols)
-        if bool_cols:
-            for name in bool_cols:
-                self.stringify_boolean_col(col_name=name)
-        if dim_rows:
-            self._dim_row_list(dim_rows)
-
-    def _table_wrapper(self, table, wrap_cols):
-        """Query for column width and wrap text"""
-        for col_name in wrap_cols:
-            index = self._get_index_for_col_name(col_name)
-            width = abs(table.column_max_width(index))
-            for row in table.table_data:
-                row[index] = (
-                    textwrap.fill(row[index], width) if row[index] is not None else ""
-                )
-
-    def stringify_boolean_col(self, col_name=None, true_str=None, false_str=None):
-        """Replace booleans in the given column name with a string.
-        Args:
-
-        * col_name: str indicating which columns should be stringifed.
-        * true_str: True values will be replaced with this string on posix systems.
-        * false_str: False values will be replaced with this string on posix systems.
-        """
-        col_index = self._get_index_for_col_name(col_name)
-
-        true_str = (
-            click.style(true_str, fg="green") if true_str else self.PICTOGRAM_TRUE
-        )
-        false_str = (
-            click.style(false_str, fg="red") if false_str else self.PICTOGRAM_FALSE
-        )
-        for row in self._table.table_data[1:]:
-            row[col_index] = true_str if row[col_index] else false_str
-
-    def echo(self, plain=False):
-        """Print this table's data using click.echo().
 
-        Automatically falls back to AsciiTable if there's an encoding error.
-        """
-        if plain or os.environ.get("TERM") == "dumb":
-            table = self.ascii_table()
+        data = [self._stringify_row(r) for r in data]
+        self._table: Table = Table(*data[0], title=title, box=box.SIMPLE, *kwargs)
+        for idx, row in enumerate(data[1:]):
+            dim_row = idx + 1 in dim_rows if dim_rows else False
+            self._table.add_row(*row, style=Style(dim=dim_row))
+
+    def _stringify_row(self, row: list) -> List[str]:
+        stringified_row = []
+        for cell in row:
+            if isinstance(cell, bool):
+                cell = self.PICTOGRAM_TRUE if cell else self.PICTOGRAM_FALSE
+            else:
+                cell = str(cell)
+            stringified_row.append(cell)
+        return stringified_row
+
+    def echo(self, plain=False, box_style: box.Box = None):
+        """Print this table's data using click.echo()."""
+        orig_box = self._table.box
+        if plain:
+            self._table.box = box.ASCII2
         else:
-            table = str(self)
-        click.echo(table)
+            self._table.box = box_style or orig_box
+        console = Console()
+        console.print(self._table)
+        self._table.box = orig_box
```

Comment:
I don't see any discernible difference when passing `--plain` to `cci org list` vs not. Small concern in the scheme of things, but am I missing something?

Reply:
I'm not sure whether the plain option still makes sense. The original use case for the option was windows and for processing a table through stdout, but Rich does the right thing when writing to stdout, and we're using `box.SIMPLE` by default, which works the same across platforms.

Reply:
We'll revisit this as part of the feedback updates.